From 9dd4db8dfeafb6464cccf81cb78038b7a252d080 Mon Sep 17 00:00:00 2001
From: jasonwryan <jasonwryan@gmail.com>
Date: Sun, 27 May 2012 12:34:27 +1200
Subject: [PATCH] Small fix to two macros in notmuch-mutt.rc

A couple of minor fixes to ensure that the two macros,
for <F9> and <F6> work as advertised.
---
 contrib/notmuch-mutt/notmuch-mutt.rc |    8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/contrib/notmuch-mutt/notmuch-mutt.rc b/contrib/notmuch-mutt/notmuch-mutt.rc
index c0ff000..38ad584 100644
--- a/contrib/notmuch-mutt/notmuch-mutt.rc
+++ b/contrib/notmuch-mutt/notmuch-mutt.rc
@@ -2,8 +2,8 @@ macro index <F8> \
       "<enter-command>unset wait_key<enter><shell-escape>notmuch-mutt --prompt search<enter><change-folder-readonly>`echo ${XDG_CACHE_HOME:-$HOME/.cache}/notmuch/mutt/results`<enter>" \
       "notmuch: search mail"
 macro index <F9> \
-      "<enter-command>unset wait_key<enter><pipe-message>notmuch-mutt thread<enter><change-folder-readonly>`echo ${XDG_CACHE_HOME:-$HOME/.cache}/notmuch/mutt/results`<enter><enter-command>set wait_key<enter>" \
-      "notmuch: reconstruct thread"
+      "<enter-command>unset wait_key<enter><enter-command>unignore message-id<enter><pipe-message>notmuch-mutt thread<enter><change-folder-readonly>`echo ${XDG_CACHE_HOME:-$HOME/.cache}/notmuch/mutt/results`<enter><enter-command>set wait_key<enter>" \
+      "search and reconstruct owning thread with notmuch"
 macro index <F6> \
-      "<enter-command>unset wait_key<enter><pipe-message>notmuch-mutt tag -inbox<enter>" \
-      "notmuch: remove message from inbox"
+      "<enter-command>unset wait_key<enter><enter-command>unignore message-id<enter><pipe-message>notmuch-mutt -- tag -inbox<enter>" \
+      "remove message from inbox with notmuch"
-- 
1.7.10.2

